cves/kernel/CVE-2022-0516.yml

```diff
@@ -235,10 +236,14 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: a43b80b782c9f56b3bcc2e5e51261dc3980839ec
+    note: | 
+      There is 2 years worth of work on this subsystem and even file from the origin
+      to the fix, but none of them really did anything with the Guest users functionality
+      except this one. This one adds a global for debugging, KVM_GUESTDBG_VALID_MASK.
+      Though it has no effect on the specific function of the bug, it is interesting that
+      permissions debugging systems were made in the bug's subsystem, and yet the bug
+      remained elusive.
```

Comment:
Do you think they were aware there might have been a bug but didn't know the exact location and couldn't replicate it? Or was this preemptive yet still didn't locate it. Either way its unfortunate they walked right over it without seeing the bug.